src/ast/query.rs

```diff
@@ -125,6 +132,26 @@ impl fmt::Display for SetOperator {
     }
 }
 
+/// A quantifier for [SetOperator].
+// TODO: Restrict parsing specific SetQuantifier in some specific dialects.
+// For example, BigQuery does not support `DISTINCT` for `EXCEPT` and `INTERSECT`
```

Comment:
Okay, I added some comments at [ae6fb81](https://github.com/sqlparser-rs/sqlparser-rs/pull/689/commits/ae6fb81c13e74ae8240016c0203a146462b9ba42) .
But, I wasn't quite sure how to describe this enum🥲